### ISSUE TITLE
Impossible migrate to AndroidX

### ISSUE THREAD

**Issue by OleksandrMovchan** [msintuneappsdk/intune-app-sdk-xamarin#48]
As we know, Google now reject all applications with API version below 29

**Impossible migrate to AndroidX**

We tried to update our target API, but for Android version starting 10 is mandatory to migrate to AndroidX library versions.
And Intune for Xamarin.Android doesn't support AndroidX, and has dependencies inside only to obsolete Android.Support.

As result, project can't be compiled due to conflict of versions.

Additionally it's impossible to migrate project to Xamarin.Forms 5 because XF5 also has dependencies on AndroidX.

**Steps to reproduce:**

1. Create new Xamarin Forms project.
2. Set target Android version to either 10 (API 29) or 11 (API 30)
3. Migrate to AndroidX if needed
3. Add dependency to Xamarin.Intune.MAM.Android and Remapper
4. Try to run project

Additionally try to use Xamarin Forms 5

**Expected behavior:**
Project is compiled. App is running.
Xamarin.Intune.MAM.Android has AndroidX dependencies for target API 29 and higher

**Comment by JamestsaiTW** [comment]
@meghandaly, Is any update news?

**Comment by nosmalldreams** [comment]
Any updates? We are trying to implement InTune support, but are running into the same issue.

**Comment by nosmalldreams** [comment]
I am trying to implement InTune for the first time in a Xamarin app already upgraded to AndroidX. I ran into the same issue as the OP when I tried to follow the Xamarin implementation instructions [here](https://docs.microsoft.com/en-us/mem/intune/developer/app-sdk-xamarin). I do see [this](https://github.com/msintuneappsdk/ms-intune-app-sdk-android/issues/12) post and the documentation [here ](https://docs.microsoft.com/en-us/intune/app-sdk-android#androidx-libraries)that indicates that for AndroidX, MAM support library references are no longer needed. However, I'm still not sure how to achieve the initialization of InTune that I think the Xamarin document is indicating would be achieved by calling IntuneMAMEnrollmentManager.Instance.LoginAndEnrollAccount([NullAllowed] string identity); Stated simply, I do not understand what the steps should be to implement InTune in Android X, as the documentation thereof seems to be obsolete.
